docs/crd_mysql_backup_schedule.md

```diff
@@ -0,0 +1,74 @@
+BackupSchedule
+==============
+
+`BackupSchedule` is a custom resource definition (CRD) that represents
+a full dump & binlog scheduling. This CR creates two `CronJob`s.
+The one is for `Dump` and the other is for `Binlog`.
+
+| Field        | Type                                          | Description                                          |
+| ------------ | --------------------------------------------- | ---------------------------------------------------- |
+| `apiVersion` | string                                        | APIVersion.                                          |
+| `kind`       | string                                        | Kind.                                                |
+| `metadata`   | [ObjectMeta]                                  | Standard object's metadata.                          |
+| `spec`       | [BackupScheduleSpec](#BackupScheduleSpec)     | Specification of scheduling.                         |
+| `status`     | [BackupScheduleStatus](#BackupScheduleStatus) | Most recently observed status of the scheduled jobs. |
+
+BackupScheduleSpec
+------------------
+
+| Field                    | Type                                    | Description                                                               |
+| ------------------------ | --------------------------------------- | ------------------------------------------------------------------------- |
+| `clusterName`            | string                                  | Name of [`Cluster`](crd_mysql_cluster.md)                                 |
+| `schedule`               | string                                  | Schedule in Cron format, this value is passed to `CronJob.spec.schedule`. |
+| `objectStorageEndpoint`  | [ObjectStorageSpec](#ObjectStorageSpec) | Specification of S3 compatible object storage.                            |
+| `retentionPeriodSeconds` | int                                     | Retention period of each backup file.                                     |
+
+BackupScheduleStatus
+--------------------
+
+| Field                | Type                                                    | Description                                    |
+| -------------------- | ------------------------------------------------------- | ---------------------------------------------- |
+| `lastCompletionTime` | [Time]                                                  | Completion time of the last backup.            |
+| `succeeded`          | boolean                                                 | `True` when the job is completed successfully. |
+| `conditions`         | [][`BackupScheduleCondition`](#BackupScheduleCondition) | The array of conditions.                       |
+
+ObjectStorageSpec
```

Comment:
It may be better for this kind to be an independent custom resource
because buckets may be used by multiple clusters.

Reply:
I'll do it!